pkg/platform/platform_test.go

```diff
@@ -45,3 +46,19 @@ func TestAppendVersionTag(t *testing.T) {
 		t.Fatalf("should append version tag to existing map")
 	}
 }
+
+func TestCommonControlPlaneChartsOrder(t *testing.T) {
+	expectedOrder := []string{"pod-checkpointer", "kube-apiserver", "kubernetes", "calico", "lokomotive", "bootstrap-secrets"} //nolint:lll
+
+	commonControlPlaneCharts := platform.CommonControlPlaneCharts()
```

Comment:
Hooking on platform.CommonControlPlaneCharts is not the ideal place to check that, as the order may still be mangled by appending to the slice or even randomized before applying, but I think it's better than nothing.

Reply:
I understand that new charts might be appended in the list. Like it is don for kubelet chart. But I don't know if the order will change as you say so. I see it is all ordered from there on.

Reply:
The point is, we do not check right before the applying, so it still can be mangled _somewhere_.